### PR TITLE
Lesson-11: added tasks

### PR DESCRIPTION
**Pull request by NPaulGuy**

### CODE REVIEW

File: Lesson11/Task1.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * 	Сделайте функцию, которая возвращает квадрат числа. Число передается параметром.
+ */
+function powDouble($num) {
```
Comment (ckomarov) [inline Lesson11/Task1.php:5]: type hint, return type

Comment (ckomarov) [reply Lesson11/Task1.php:5]: function powDouble(int $num) : int { ...

File: Lesson11/Task2.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Сделайте функцию, которая возвращает сумму двух чисел. Числа передаются параметрами функции. 
+ */
+function numbersSum($num1, $num2) {
```
Comment (ckomarov) [inline Lesson11/Task2.php:5]: same as above

File: Lesson11/Task1.php (hunk 1 above)
Comment (ckomarov) [reply Lesson11/Task1.php:5]: check php7 features

File: Lesson11/Task3.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * 	Сделайте функцию, которая отнимает от первого числа второе и делит на третье.
+ */
+function minusAndDivide($num, $minusNum, $divNum) {
```
Comment (ckomarov) [inline Lesson11/Task3.php:5]: same

File: Lesson11/Task4.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Сделайте функцию, которая принимает параметром число от 1 до 7, а возвращает день недели на русском языке.
+ */
+function weekDay($weekDayNumber) {
```
Comment (ckomarov) [inline Lesson11/Task4.php:5]: same + add phpdoc to all functions pls

**Review by ckomarov — CHANGES_REQUESTED** [review]

File: Lesson11/Task4.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    $week = [1 => 'понедельник', 'вторник','среда','четверг','пятница', 'суббота', 'воскресенье'];
+    if ($weekDayNumber >= 1 && $weekDayNumber <= 7) {
+        return $week[$weekDayNumber];
+    } else {
+        return false;
```
Comment (ckomarov) [inline Lesson11/Task4.php:10]: better to have just one return type in function

Comment (ckomarov) [reply Lesson11/Task4.php:10]: so change false to 0 pls
0 == false

**Review by ckomarov — CHANGES_REQUESTED** [review]

File: Lesson11/Task4.php (hunk 5 above)
Comment (NPaulGuy) [reply Lesson11/Task4.php:10]: But the return type must be string, not int: $week array has string names of weekDays. It may be done like "" == false.

**Review by ckomarov — APPROVED** [review]

**Pull request merged**